### ISSUE TITLE
[ENHANCE] De-mat Device (sort of)

### ISSUE THREAD

**Issue by Jwtc2000** [DarkholmeTenk/TardisModIssues#115]

**Comment by Soyaro666** [comment]
just trying to understand your example...
if I place my Tardis in front of your entrance door you could turn my tardis into uncoordinated flight to walk through it...?
it's an interesting idea but not really better than a Tardis blocking someones door. if a Tardis is in any flight mode nobody can leave or enter it... or did I get that wrong?

**Comment by FoxPotato** [comment]
Correct me if I'm wrong, but I think he means that when someone uses this gun on a TARDIS it dematerializes and then after a bit rematerializes. So he can get to whatever the TARDIS was blocking.
And didn't another user already suggest this.

**Comment by Soyaro666** [comment]
yes, of course he means the Tardis will return after a short moment. what I meaned is: while it is dematerialized nobody can leave or enter it (well, from the inside you would at least have a chance to react) and if a player exists who griefs other players with a Tardis, then another player (or even the same player, that doesn't really matter) could exist who just stays there and keeps fireing at a Tardis to grief it's owner.
I don't think fixing players misbehave should be the job for a mod builder... :-/

**Comment by FoxPotato** [comment]
Like I said, someone else already suggested this and Dark said no to that.

**Comment by DarkholmeTenk** [comment]
With the use case of "I need to get to something that someone's blocking", a single use shunting device which just forces the TARDIS to rematerialise somewhere nearby may be something I'd implement.

**Comment by Jwtc2000** [comment]
/\ :+1:

**Comment by FoxPotato** [comment]
It'd be a better function than destroying a TARDIS when it's de materializing
